lib/repl.js

```diff
@@ -1195,3 +1195,4 @@ function Recoverable(err) {
   this.err = err;
 }
 inherits(Recoverable, SyntaxError);
+exports.Recoverable = Recoverable;
```

Comment:
@blakeembrey I suggest to export `isRecoverableError` function as well so that user can make use of it.

CC:
@thefourtheye